config/freertos/crazyflie21/board.repos

```diff
@@ -1,8 +1,8 @@
 repositories:
   crazyflie_firmware:
     type: git
-    url: https://github.com/bitcraze/crazyflie-firmware
-    version: 2020.06
+    url: https://github.com/Acuadros95/crazyflie-firmware
+    version: feature/link_tx_info
```

Comment:
```suggestion
    url: https://github.com/bitcraze/crazyflie-firmware
    version: 2021.06
```

Modify if this PR https://github.com/bitcraze/crazyflie-firmware/pull/852 gets merged.